docs/Using-Fleet/REST-API.md

```diff
@@ -1717,27 +1699,32 @@ None.
-| software_id             | integer | query | The ID of the software to filter hosts by.                                                                                                                                                                                                                                                                                                  |
+| Name                    | Type    | In    | Description                                                                                                                                                                                                                                                                                                                                |
+| ----------------------- | ------- | ----- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| page                    | integer | query | Page number of the results to fetch.                                                                                                                                                                                                                                                                                                       |
+| per_page                | integer | query | Results per page.                                                                                                                                                                                                                                                                                                                          |
+| order_key               | string  | query | What to order results by. Can be any column in the hosts table.                                                                                                                                                                                                                                                                            |
+| after                   | string  | query | The value to get results after. This needs order_key defined, as that's the column that would be used.                                                                                                                                                                                                                                     |
+| order_direction         | string  | query | **
```

Comment:
I don't think this will render correctly with the newline after `**`. You shouldn't change the lines that did not require modifications, as it makes it harder to review (my guess is that your editor automatically cuts lines beyond a certain length?).

Reply:
Oops. I must have accidentally enabled auto-reformatting on write. Same happened to the schema.sql before.